### ISSUE TITLE
Make “Synchronization strategy” profile global

### ISSUE THREAD

**Issue by cnpym-github** [floccusaddon/floccus#1809]
### Describe the feature you'd like to request

Make “Synchronization strategy” profile global.
Show only “Pull down once” or “Push up once” according to “Synchronization strategy”.
“Sync now” obeys “Synchronization strategy”.

### Describe the solution you'd like

Make “Synchronization strategy” profile global.
Show only “Pull down once” or “Push up once” according to “Synchronization strategy”.
“Sync now” obeys “Synchronization strategy”.

### Describe alternatives you've considered

none

**Comment by marcelklehr** [comment]
Hi @cnpym-github 

Can elaborate a bit on this? Currently "Sync now" should already obey the sync strategy setting.

**Comment by cnpym-github** [comment]
I'm terribly afraid to mistakenly hit pull or push

**Comment by marcelklehr** [comment]
Yes, it should. It might not, that would be a bug, then. But it shoud. I wrote *should* because you seem to indicate it doesn't for you.

**Comment by cnpym-github** [comment]
how about option to hide pull and push?

**Comment by marcelklehr** [comment]
> how about option to hide pull and push?

The pull and push buttons are there for convenience, I don't think hiding them is the right move. What may be useful for you to know: You should not need to manually hit sync in normal operation if you enable Auto-sync in the settings. This will automatically sync your bookmarks every X minutes and directly after a local change.
